packages/amplify-cli/src/domain/amplify-telemetry/getTelemetryUrl.ts

```diff
@@ -0,0 +1,10 @@
+import url, { UrlWithStringQuery } from 'url';
+import { getLatestApiVersion } from './VersionManager';
+
+const version = getLatestApiVersion();
+const prodUrl = `https://aws-amplify-cli-telemetry.us-east-1.amazonaws.com/${version}/metrics`;
```

Comment:
It seems that the payload version and the API version for telemetry is coupled together?

Reply:
Yes the payload version is coupled together with the API version